lsaneo/cmd/config.yaml

```diff
@@ -0,0 +1,14 @@
+termMappings:
+  https://lschema.org/AttributeName: ls:attributeName
+  https://lschema.org/AttributeValue: ls:attributeValue
+  https://lschema.org/EntityID: ls:entityId
+  https://lschema.org/NodeID: ls:nodeId
+  https://lschema.org/SchemaNodeID: ls:schemaNodeId
+  https://lschema.org/EdgeLabel: ls:edgeLabel
+
+namespaceMappings:
+  https://lschema.org/: ls
+  https://lschema.org/A: ls:A
```

Comment:
not ls:A, but lsa, i.e. no colon in namespace mappings

  https://lschema.org/A/: lsa

so it maps: https://lschema.org/A/b to lsa:b


trie.search would return this:
search(https://lschema.org/A/b) (https://lschema.org/A/, lsa, true)
Meaning: replace ret[0] in the input string with ret[1].